test/models/user_test.rb

```diff
@@ -26,6 +26,12 @@ def setup
     refute User.new(email: test_email).valid?
   end
 
+  test 'email must be saved downcased' do
+    new_email = 'NEW_EMAIL@exaMple.cOm'
+    assert create(:user, email: new_email.downcase)
+    refute User.new(email: new_email).valid?
```

Comment:
I think you can remove this test as it's not really testing the validity of down casing the email. You can test this out by commenting out your newly added `before_save` filter. If email case was affecting this test you would expect it to fail.